Apps/phvxstream/__init__.py

```diff
@@ -0,0 +1,9 @@
+# File: __init__.py
+# Copyright (c) 2021 Splunk Inc.
```

Comment:
The copyright is generated from running this script: https://cd.splunkdev.com/phantom/appscript/-/blob/github/update_cr.py